horaemeta/Dockerfile

```diff
@@ -44,7 +44,9 @@ RUN chmod +x /usr/bin/horaemeta-server
 COPY ./docker/entrypoint.sh /entrypoint.sh
 COPY ./config/example-standalone.toml /etc/horaemeta/horaemeta.toml
 
-COPY ./docker/tini /tini
+ENV TINI_VERSION v0.19.0
```

Comment:
Maybe we should move this line after L19.
BTW, I guess `ARG` command should work better than `ENV` because `TINI_VERSION` is not necessary to be known by the started container.